scripts/test/testQA.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { resolve } from 'path'
+import { runTests } from '@vscode/test-electron'
+import { setupVSCodeTestInstance } from './launchTestUtilities'
+;(async () => {
+    try {
+        console.log('Running Main test suite...')
+
+        const vsCodeExecutablePath = await setupVSCodeTestInstance()
+        const rootDir = process.cwd()
+        const testEntrypoint = resolve(rootDir, 'dist/src/testQA/index.js')
+        const testWorkspace = resolve(rootDir, 'src/testFixtures/workspaceFolder')
+        console.log(`Starting tests: ${testEntrypoint}`)
+
+        const result = await runTests({
+            vscodeExecutablePath: vsCodeExecutablePath,
```

Comment:
Not sure we want the lint job to be driven by the mocha test runner. When we later add linting, that will be driven by eslint. Invoking that from mocha seems like too many layers. It also requires starting vscode, which is slow.

Instead this "test" could live in `scripts/lint/`. Later we may add more lint rules there. That makes linting faster than invoking via mocha+vscode. 

Are there advantages to using the test runner? 

- nice reporting
- common structure via it(), describe()
- ...?

Reply:
I updated this so that we don't used the vscode test runner, instead running directly through the Mocha runner which is instantaneous.

The advantages I see are what you listed. It matches the same format as all our other tests, so easy to understand.
- Any necessary setup can be easily done and is isolated to that test (eg: setting up fake files to test against)